packages/langium-cli/src/langium.ts

```diff
@@ -20,12 +20,20 @@ program
     .command('generate')
     .description('Generate code for a Langium grammar')
     .option('-f, --file <file>', 'the configuration file or package.json setting up the generator')
+    .option('-t, --types <file>', 'the langium grammar file to generate types for')
```

Comment:
Thanks! I'd propose to define a new command instead of an option for the existing command. What about `extract-types`?

Reply:
I had problems with adding CLI options with `-` in the name. I will try it with `-e, --extract-types` again

Reply:
I mean defining a command, not an option:
```typescript
    .command('extract-types')
    .action(() => ...)
```

Reply:
Oh you mean like `langium extract-types -g mygrammar.langium --target types.langium` ?

Reply:
Yes, something like that. Either for a specific grammar given as option or for all grammars mentioned in langium-config.json

Reply:
I've added a new command `extract-types`.
Usage: `langium extract-types <file> [--output <file>]`

Generating types for multiple grammars in one flow can be added later